build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj

```diff
@@ -19,4 +19,28 @@
     <ProjectReference Include="..\..\external\xamarin-android-tools\src\Xamarin.Android.Tools.AndroidSdk\Xamarin.Android.Tools.AndroidSdk.csproj" />
   </ItemGroup>
 
+ <!-- See https://github.com/xamarin/xamarin-android/commit/ceca993068c0401f664676db85edec1d772bf5ff -->
+  <ItemGroup>
+    <None Include="$(PkgMono_Unix)\runtimes\linux-x64\native\libMono.Unix.so" CopyToOutputDirectory="PreserveNewest" />
+    <_MonoUnixDylib Include="$(PkgMono_Unix)\runtimes\osx-arm64\native\libMono.Unix.dylib" Condition=" '$(HostOS)' == 'Darwin' " />
+    <_MonoUnixDylib Include="$(PkgMono_Unix)\runtimes\osx-x64\native\libMono.Unix.dylib" />
+  </ItemGroup>
+  <PropertyGroup>
+    <_MonoUnixOutputDylib>$(OutputPath)$(TargetFrameworkNETStandard)\libMono.Unix.dylib</_MonoUnixOutputDylib>
+  </PropertyGroup>
+  <Target Name="_MakeMonoUnixFatBinariesOSX"
+      AfterTargets="CopyFilesToOutputDirectory"
+      Inputs="@(_MonoUnixDylib)"
+      Outputs="$(_MonoUnixOutputDylib)">
+    <Exec
+        Command="lipo @(_MonoUnixDylib->'&quot;%(FullPath)&quot;', ' ') -create -output &quot;$(_MonoUnixOutputDylib)&quot;"
+        Condition=" '$(HostOS)' == 'Darwin' "
+    />
+    <Copy
+        SourceFiles="@(_MonoUnixDylib)"
+        DestinationFolder="$(OutputPath)"
+        Condition=" '$(HostOS)' != 'Darwin' "
+    />
+  </Target>
```

Comment:
Seems like we have this twice now:

https://github.com/xamarin/xamarin-android/blob/fc12409a0690ceea491be8f0a05890972da70860/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets#L373-L388

Is there a way to put this target in its own file and import twice? If we used the same MSBuild properties?

Reply:
This was my original thought, but the overhead didn't seem worth it compared to copying a few lines. I think we'd want this if we need to do something similar in a third or fourth place in the future though, so I will try to adjust this.